site/_guides/gateway-api.md

```diff
@@ -0,0 +1,118 @@
+evolve service networking APIs within the Kubernetes ecosystem. Gateway API consists of multiple resources that provide
+user interfaces to expose Kubernetes applications- Services, Ingress, and more.
+
+### Background
+
+Gateway API targets three personas:
+
+- __Platform Provider__: The Platform Provider is responsible for the overall environment that the cluster runs in, i.e.
+  the cloud provider. The Platform Provider will interact with GatewayClass and Contour resources.
+- __Platform Operator__: The Platform Operator is responsible for overall cluster administration. They manage policies,
+  network access, application permissions and will interact with the Gateway resource.
+- __Service Operator__: The Service Operator is responsible for defining application configuration and service
+  composition. They will interact with xRoute resources and other typical Kubernetes resources.
+
+Gateway API contains three primary resources:
+
+- __GatewayClass__: Defines a set of gateways with a common configuration and behavior.
+- __Gateway__: Requests a point where traffic can be translated to a Service within the cluster.
+- __xRoutes__: Describes how traffic coming via the Gateway maps to the Services.
+
+Resources are meant to align with personas. For example, a platform operator will create a Gateway, so a developer can
+expose an HTTP application using an HTTPRoute resource.
+
+### Prerequisites
+The following prerequisites must be met before using Gateway API with Contour:
+
+- A working [Kubernetes][2] cluster. Refer to the [compatibility matrix][3] for cluster version requirements.
+- The [kubectl][4] command-line tool, installed and configured to access your cluster.
+
+### Using Gateway API with Contour Operator
+
+The preferred method for using Gateway API is with [Contour Operator][5]. This is because the GatewayClass and Gateway
+resources should be managed by active controllers that are implemented by the operator. Refer to the [contour][6] and
+[operator][7] designs for additional background on the gateway API implementation.
+
+Run the operator:
+```shell
+$ kubectl apply -f https://projectcontour.io/quickstart/operator.yaml
+```
+This command creates:
+
+- Namespace `contour-operator` to run the operator.
+- Operator and Contour CRDs.
+- Operator RBAC resources for the operator.
+- A Deployment to manage the operator.
+- A Service to front-end the operator’s metrics endpoint.
+
+Create the Gateway API resources:
+```shell
+$ kubectl apply -f https://projectcontour.io/quickstart/gateway.yaml
+```
+This command creates:
+
+- Namespace `projectcontour` to run the Gateway and child resources, i.e. Envoy DaemonSet.
+- A Contour custom resource named `contour-gateway-sample` in the operator's namespace. This resource exposes
+  infrastructure-specific configuration and is referenced by the GatewayClass.
+- A GatewayClass named `sample-gatewayclass` that abstracts the infrastructure-specific configuration from Gateways.
```

Comment:
Maybe we should tie the name of the class to Contour in some way? No big deal either way just might help identify it better.